builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -64,7 +64,7 @@ func TestAccVSphereVirtualMachine_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(
```

Comment:
Negative tests cases are AWESOME!!!!  Break it ... see what happens when it goes boom.  You have coverage so we are ok, but in the future.